### PR TITLE
New package: Satty

### PR DESCRIPTION
**Pull request by RotaryBoot58**
#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO** Yes
Yes

### CODE REVIEW

File: srcpkgs/satty/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'satty'
+pkgname=satty
+version=0.12.0
+revision=1
+build_style=cargo
+hostmakedepends="glib-devel pkg-config libadwaita-devel gtk4-devel gdk-pixbuf-devel libepoxy-devel fontconfig"
+makedepends="glib libadwaita gtk4 gdk-pixbuf fontconfig libepoxy"
+depends=" glib libadwaita gtk4 gdk-pixbuf fontconfig libepoxy"
```
Comment (Clos3y) [inline srcpkgs/satty/template:8]: You don't need this line, xbps automatically detects these runtime dependencies

Comment (RotaryBoot58) [reply srcpkgs/satty/template:8]: Oh ok, did not know that

Comment (Clos3y) [reply srcpkgs/satty/template]: Oh no no, sorry, I don't know why my comment referenced lines 5-8: I wrote the comment only referring to line 8. You do need lines 5-7, xbps doesn't pull in make depends but does pull in runtime depends. Apologies for the confusion.

Comment (RotaryBoot58) [reply srcpkgs/satty/template]: No worries. I am fixing it right up!

**Comment by RotaryBoot58** [comment]
I messes up some commits so I am going to open a new one

**Pull request closed without merging**